src/_pytest/fixtures.py

```diff
@@ -899,7 +899,7 @@ def execute(self, request):
         return hook.pytest_fixture_setup(fixturedef=self, request=request)
 
     def __repr__(self):
-        return "<FixtureDef name=%r scope=%r baseid=%r >" % (
+        return "<FixtureDef name=%r scope=%r baseid=%r>" % (
             self.argname,
             self.scope,
             self.baseid,
```

Comment:
Might also have been good to replace "name" with "argname"?

Reply:
Hmm yes I think so